tests/classes/opengraph-test.php

```diff
@@ -17,10 +17,10 @@ class OpenGraph_Test extends TestCase {
 	/**
 	 * Test that our constructor works.
 	 *
-	 * @covers WPSEO_WooCommerce_OpenGraph::__construct
+	 * @covers OpenGraph::__construct
```

Comment:
`@covers` tags MUST be fully qualified names. See: https://phpunit.readthedocs.io/en/7.5/annotations.html#covers

```suggestion
	 * @covers \Yoast\WP\Woocommerce\Classes\OpenGraph::__construct
```

(same goes for all the other `@covers` tag adjustments in this and the other files.)